service/controller/v26/cloudconfig/master_template.go

```diff
@@ -225,7 +225,6 @@ ExecStart=/bin/sh -c '\
 		sleep 1m && \
 		/usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /etc/kubernetes:/etc/kubernetes $KUBECTL -n kube-system delete pod -l k8s-app=kube-proxy && \
 		/usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /etc/kubernetes:/etc/kubernetes $KUBECTL -n kube-system delete pod -l k8s-app=calico-kube-controllers && \
-		/usr/bin/docker run -e KUBECONFIG=${KUBECONFIG} --net=host --rm -v /etc/kubernetes:/etc/kubernetes $KUBECTL -n kube-system delete pod -l k8s-app=coredns &&\
 		result="ok" || echo "failed";\
 	done;\
 	[ "$result" != "ok" ] && echo "Failed to restart k8s services." && exit 1 || echo "Successfully restarted k8s services.";'
```

Comment:
Can we check if this `exit 1` statement actually gets hit now? Can we somehow make it simpler to ensure we fail? e.g. by simplifying the expression here?

Reply:
I think it's not an error using `kubectl delete pod -l k8s-app=coredns` even if no pod matches that label. I will confirm though.